.github/workflows/build-images.yml

```diff
@@ -364,7 +364,19 @@ jobs:
         else
           make docker-build-operator
         fi
-
+    - name: Optionally push operator
+      if: ${{ github.event_name != 'pull_request' && inputs.operator_image == '' }}
```

Comment:
This case has been done in line 363. For non-pr e2e tests, we already push the image to Github repo. Just move your image-exists check mechanism to that line.

Reply:
I moved the mechanism in https://github.com/vertica/vertica-kubernetes/pull/928/commits/1b99dc69afddfa8cdec57cab72316a1f9d5ddc3a
For the image check mechanism, we need to do the docker login first, to grant access to check the manifest. I was thinking doing that in the make file, but the credentials are stored in the git env, so keeping in the yml for now. I am open to other approaches though.